contracts/translation/translation.go

```diff
@@ -0,0 +1,24 @@
+package translation
+
+//go:generate mockery --name=Translation
+type Translation interface {
+	// Load translations from path.
+	Load(path string, locale ...string) error
+	GetDefaultLocale() string
```

Comment:
S: consistent with Laravel: https://github.com/laravel/framework/blob/c262c1ffce93c5fa4b9e00da1a54d8c02ddddcf6/src/Illuminate/Translation/Translator.php#L404
```suggestion
	Locale() string
```